.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -133,11 +133,8 @@ jobs:
           name: web-test-runner-debugging-screenshots
           path: dist/web-test-runner-debugging-screenshots
 
-  # Its own job instead of putting it in `test-visuals` and renaming that job
-  # because `test-visual` sets `continue-on-error` so that missing and changed
-  # snapshots don't cause it to fail.
-  test-aria:
-    name: Test (ARIA)
+  test-functionality:
+    name: Test (Functionality)
```

Comment:
Let me know what you think. But the plan is to put every other non-visual, non-lint rule test into this project. Three projects altogether:

<img width="974" height="1980" alt="image" src="https://github.com/user-attachments/assets/c819526a-858d-41fa-93d0-ecbd68b8f515" />